iceoryx_posh/include/iceoryx_posh/internal/popo/rpc_interface.hpp

```diff
@@ -32,6 +32,8 @@ class RpcInterface
   public:
     /// @brief Sends the given Request<T> or Response<T> via the type which implements this interface
     /// @param[in] rpcData is the actual Request<T> or Response<T> instance
+    /// @todo iox-#27 for Request<T> the return value must be a `cxx::expected<ClientSendError>` and for
+    /// Response<T> a `cxx::expected<ServerSendError>`
```

Comment:
The tests for the typed API are not done yet and the example is not in master, so I cannot easily test the SFINEA which is necessary. I will change this with the test for the typed API

Reply:
Wasn't that difficult after all